<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Development Rail Crack of Railway</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>minilogo.png</normaloff>minilogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="capture">
     <attribute name="title">
      <string>Capture</string>
     </attribute>
     <widget class="QPushButton" name="capture_Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_Button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>681</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>681</width>
        <height>351</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="raw">
       <attribute name="title">
        <string>Raw Data</string>
       </attribute>
       <widget class="QListWidget" name="data_raw">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>471</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="b_xlsx">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>181</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Save .xlsx</string>
        </property>
       </widget>
       <widget class="QPushButton" name="b_csv">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>130</y>
          <width>181</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Save .csv</string>
        </property>
       </widget>
       <widget class="QPushButton" name="b_txt">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>220</y>
          <width>181</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>Save .txt</string>
        </property>
       </widget>
       <widget class="QLabel" name="save_name">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>531</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Latest file : </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="graph">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <widget class="SIGNAL" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>651</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>FFT</string>
       </attribute>
       <widget class="SIGNAL" name="widget_4" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>651</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>STFT</string>
       </attribute>
       <widget class="QLabel" name="spec_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>651</width>
          <height>301</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>คลิกเพื่อขยาย</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="analysis">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>command</string>
      </property>
      <widget class="QPushButton" name="load_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>load</string>
       </property>
      </widget>
      <widget class="QPushButton" name="analysis_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>analysis</string>
       </property>
      </widget>
      <widget class="QLabel" name="load_name">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>461</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Latest file : </string>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Signal</string>
       </attribute>
       <widget class="SIGNAL" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>FFT</string>
       </attribute>
       <widget class="SIGNAL" name="widget_3" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>STFT</string>
       </attribute>
       <widget class="QLabel" name="spec">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>641</width>
          <height>281</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>คลิกเพื่อขยาย</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QComboBox" name="port_combobox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>30</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Development Rail Crack of Railway&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIGNAL</class>
   <extends>QWidget</extends>
   <header>SIGNAL.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
